core/utils/toolbox.js

```diff
@@ -0,0 +1,158 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Utility functions for the toolbox and flyout.
+ * @author aschmiedt@google.com (Abby Schmiedt)
+ */
+'use strict';
+
+goog.provide('Blockly.utils.toolbox');
+
+
+/**
+ * A block definition.
+ * @typedef {{
+ *            contenttype:string,
```

Comment:
camelcase?

Reply:
I assume you don't want to go with type here as it could be confused with a block's type.
How about tag? or kind?